authorization/jwt-signing.md

```diff
@@ -124,3 +129,45 @@ The following example contains every single option available:
     }
   ]
 {{< /highlight >}}
+
+## Example: How to generate a JWT token
+The following example gives a quick start to adopt JWT in your backend. Essentially, what you need to do is to adapt your existing `/login` function (maybe passing an additional `?token=true` flag), so when a user logs in, instead of setting the session in a cookie, you return the JSON Web Token for KrakenD to sign. 
+
+The token is no more than a JSON output adhering to the [JWT standard](https://tools.ietf.org/html/rfc7519).
+
+The following code (PHP) in your backend would suffice to let KrakenD do the signing:
```

Comment:
I think we should add examples using other languages if we are going to show actual code... the latest TIOBE list shows PHP ranking 8th (maybe adding JS, python and go could be enough)